### PR TITLE
Category docs

### PR DESCRIPTION
**Pull request by Juuddi**
- Add category documentation pages (some with content, some without)
- Transition `basics/abi.md` to `static/abi.mdx`
- Remove arbitrary flatwork mdx wrap to standardize markdown formatting
- Remove `docs/` from `.prettierignore` for formatting on commit

### CODE REVIEW

File: docs/contributing.md [hunk 1]
```diff
@@ -2,69 +2,49 @@
...
+Contributing to the documentation is welcome, but when making changes to documentation, please ensure that all changes are made **only** to:
 
-- Updating `/docs/*\*.md`
-- Updating API jsdocs: `/*\* ... */` comment blocks within `/src/`
+-   Updating `/docs/*\*.md`
+-   Updating API jsdocs: `/*\* ... */` comment blocks within `/src/`
 
-When adding a new sections, a [[link-discussion]] is
-preferred.
+When adding a new sections, a [quais Discussion](https://github.com/quais-io/quais.js/discussions) is preferred.
```
Comment (rileystephens28) [inline docs/contributing.md:24]: The link to quais repo is wrong. Should be under ds org

**Comment by rileystephens28** [comment]
There are a few mentions of Ethereum things that we can remove later so it's okay that they are in here for now. I am also not sure whether we are still compliant with EIP191 since it is based on RLP encoding which we no longer use. We can give these a more thorough audit in the near future.

File: README.md [hunk 2]
```diff
@@ -1,62 +1,47 @@
...
+-   Extensive documentation **coming soon**
+-   Large collection of **test cases** which are maintained and added to
+-   Fully written in **TypeScript**, with strict types for security and safety
+-   **MIT License** (including ALL dependencies); completely open source to do with as you please
 
 ## Keep Updated
 
-For advisories and important notices, follow [@quaisproject](https://twitter.com/quaisproject)
-on Twitter (low-traffic, non-marketing, important information only) as well as watch this GitHub project.
+For advisories and important notices, follow [@quainetwork](https://twitter.com/quaisnetwork)
```
Comment (rileystephens28) [inline README.md:28]: `https://twitter.com/quaisnetwork` should be `https://twitter.com/quainetwork`

**Review by rileystephens28 — COMMENTED** [review]
Other than that typo in the twitter url everything looks good.

File: README.md (hunk 2 above)
Comment (Juuddi) [reply README.md]: Ah yes thx, fixed here: [886e374](https://github.com/dominant-strategies/quais-6.js/pull/167/commits/886e374e04bb82efd2b66cbd33f07bc1a03c3edb)

**Review by rileystephens28 — APPROVED** [review]

**Pull request merged**